CTP/shell/src/com/navercorp/cubridqa/shell/main/GeneralLocalTest.java

```diff
@@ -15,15 +15,15 @@
 
 	public GeneralLocalTest(Context context) {
 		this.context = context;
-		this.context.setLogDir(context.getProperty("main.testing.category", "general"));
-		String feedbackType = context.getProperty("main.feedback.type", "").trim();
+		this.context.setLogDir(context.getTestCategory());
+		String feedbackType = context.getFeedbackType();
 		if (feedbackType.equalsIgnoreCase("file")) {
 			context.setFeedback(new FeedbackFile(context));
 		} else if (feedbackType.equalsIgnoreCase("database")) {
 			context.setFeedback(new FeedbackDB(context));
 		}
-		context.setTestBuild(context.getProperty("main.testing.build_id"));
```

Comment:
It's better to rollback and keep no changes because main.testing.build_id and main.testing.bits are extended parameters.
-		context.setTestBuild(context.getProperty("main.testing.build_id"));
 -		context.setVersion(context.getProperty("main.testing.bits"));
 +		context.setTestBuild(context.getTestBuild());   <---- null value
 +		context.setVersion(context.getVersion());         <-----null value